seatunnel-core/seatunnel-flink-starter/pom.xml

```diff
@@ -103,6 +103,17 @@
             <artifactId>junit</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.seatunnel</groupId>
+            <artifactId>connector-jdbc</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.postgresql</groupId>
```

Comment:
If necessary, I recommend putting it in the JDBC dependency, and PG will be the default implementation we provide.

Reply:
If put it in the pom of jdbc connector, it doesn't work because the driver could not be found.

Reply:
did I miss something?  could you try it?

Reply:
I have tried it.

Reply:
https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-connectors-v2/connector-jdbc/pom.xml#L49 We will use it as the default driver, so we need to change its scope